app/src/main/java/org/mozilla/fenix/settings/creditcards/CreditCardEditorFragment.kt

```diff
@@ -63,8 +65,23 @@ class CreditCardEditorFragment : SecureFragment(R.layout.fragment_credit_card_ed
         creditCardEditorView.bind(creditCardEditorState)
     }
 
+    /**
+     * Close any open dialogs or menus and reauthenticate if the fragment is paused and
+     * the user is not navigating to [CreditCardsManagementFragment].
+     */
+    override fun onPause() {
+        menu.close()
```

Comment:
Wondering about the reasoning behind this? Since there's no context menu or open dialog here, I don't think we are closing anything here.

Reply:
There is an options menu and it has the delete button in it. This menu.close() prevents a visual bug that we found in logins.